.travis.yml

```diff
@@ -1,8 +1,15 @@
 sudo: false
 
+language: node_js
+node_js:
+  - node # latest stable
+
 cache:
   directories:
-    - tests/elm-stuff/build-artifacts
+    - node_modules
+    - elm-stuff
+    - tests/elm-stuff
+    - styleguide-app/elm-stuff
```

Comment:
checks out.

how do we clear the cache? We've had issues with `elm-stuff` being corrupted before.  Curious about how we fix that problem.

Reply:
https://travis-ci.org/NoRedInk/noredink-ui/caches (find it under the build page -> more options -> caches)

you can just drop 'em there with a click. Anyone who's a repo owner can do that AFAICT.